packages/core/src/WorkflowExecute.ts

```diff
@@ -154,8 +154,12 @@ export class WorkflowExecute {
 		return this.processRunExecutionData(workflow);
 	}
 
-	static isAbortError(e?: ExecutionBaseError) {
-		return e?.message === 'AbortError';
+	static isAbortError(e?: Error) {
+		return (
+			e?.name === 'AbortError' ||
+			(e instanceof WorkflowOperationError &&
+				e.message === 'Workflow has been canceled or timed out')
```

Comment:
We should create a new error type, use that on [this line](https://github.com/n8n-io/n8n/blob/master/packages/core/src/WorkflowExecute.ts), and then update this check to not use `e.message`.